From: Joel Savitz <jsavitz@redhat.com>
Subject: selftests/vm: add CATEGORY for ksm_functional_tests
Date: Thu, 27 Oct 2022 13:00:43 -0400

commit ("selftests/vm: add KSM unmerge tests") in linux-next adds an entry
to run_vmtests.sh.  I recently submitted commit b5ba705c2608
("selftests/vm: enable running select groups of tests") to linux-next
which categorizes tests by functionality in order to allow more precise
selection of which tests are to be run.

Since this newest test targets ksm and does not require more than one numa
node, add 'CATEGORY="ksm"' to the invocation to group this test with the
other ksm tests.

Link: https://lkml.kernel.org/r/20221027170043.2363797-1-jsavitz@redhat.com
Signed-off-by: Joel Savitz <jsavitz@redhat.com>
Cc: Joel Savitz <jsavitz@redhat.com>
Cc: Nico Pache <npache@redhat.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/tools/testing/selftests/vm/run_vmtests.sh~selftests-vm-add-ksm-unmerge-tests-fix
+++ a/tools/testing/selftests/vm/run_vmtests.sh
@@ -252,7 +252,7 @@ CATEGORY="ksm_numa" run_test ./ksm_tests
 # KSM test with 2 NUMA nodes and merge_across_nodes = 0
 CATEGORY="ksm_numa" run_test ./ksm_tests -N -m 0
 
-run_test ./ksm_functional_tests
+CATEGORY="ksm" run_test ./ksm_functional_tests
 
 # protection_keys tests
 if [ -x ./protection_keys_32 ]
_
